src/templates/subscription/folding.html

```diff
@@ -0,0 +1,19 @@
+<a class="pf-widget-caption">
+	<p class="pf-widget-headline"></p>
+	<span>&rsaquo;</span>
+</a>
+<a class="pf-widget-caption-left">
+	<p class="pf-widget-headline"></p>
+	<span>&rsaquo;</span>
+</a>
+<div class="pf-widget-body">
+</div>
+<div class="pf-widget-content">
+	<p class="pf-widget-message"></p>
+	<form>
+		<button type="submit" class="pf-widget-btn pf-widget-ok">X</button>
```

Comment:
Also kind of weird that it defaults to "X" as the text for the submit button. I know it gets overwritten in the js, but ??? Again maybe there should be a separate pr where we uniform all the templates. I like how I'm noticing this all now that they're actually readable. 😆 👏